### PR TITLE
Features/rhpm ql01

### PR DESCRIPTION
**Pull request by bharathkeerthi**
qualification screen and service method

### CODE REVIEW

File: web/src/app/admin/qualifications/qualifications.component.ts [hunk 1]
```diff
@@ -1,14 +1,61 @@
...
 })
 
 export class QualificationsComponent implements OnInit {
-  constructor() {
-  }
+ qualificationsModel: QualificationViewModel = {} as QualificationViewModel;
+  qualifications: QualificationViewModel[] = [] as QualificationViewModel[];
+  isCreateOrUpdate: boolean = true;
 
+  constructor(private QualificationsServiceApp: QualificationsServiceApp) {
```
Comment (salilsaini868) [inline web/src/app/admin/qualifications/qualifications.component.ts]: use camel casing for variable declarations everywhere

**Review by salilsaini868 — CHANGES_REQUESTED** [review]

File: web/src/app/admin/skills/skills.component.html [hunk 2]
```diff
@@ -1 +1,64 @@
-<p>skills</p>
+<div class="container">
+    <div class="panel panel-primary">
+        <div id="isCreate" class="panel-heading">
+            {{ 'SKILLS.CREATE_SKILLS' | translate }}
+        </div>
+        <div *ngIf="isUpdate" id="updatePanel" class="panel-heading">
+            {{ 'SKILLS.UPDATE_SKILLS' | translate }}
+        </div>
```
Comment (salilsaini868) [inline web/src/app/admin/skills/skills.component.html]: why these conditions are here again?

File: web/src/app/admin/skills/skills.component.ts [hunk 3]
```diff
@@ -1,14 +1,95 @@
...
-  constructor() {
-  }
 
+  skillsModel: SkillViewModel = {} as SkillViewModel;
+  skills: SkillViewModel[] = [];
+  isCreate: boolean = false;
+  isUpdate: boolean = false;
+  
+  isSkillExists: boolean = false;
+
```
Comment (salilsaini868) [inline web/src/app/admin/skills/skills.component.ts]: why these conditions are here again?

File: web/src/app/admin/skills/skills.component.ts [hunk 4]
```diff
@@ -1,14 +1,95 @@
...
+  }
+  msg: any = "";
+
+  onSubmit(skillsform) {
+    if (skillsform.valid) {
+      if (this.skillsModel['skillId'] === undefined) {
+        this.SkillsServiceApp.addSkill(this.skillsModel).subscribe(
+          (data) => {
+            this.skills.push(data.body);
+            this.msg = " created successfully";
```
Comment (salilsaini868) [inline web/src/app/admin/skills/skills.component.ts]: why these conditions are here again?

File: web/src/app/admin/skills/skills.component.ts [hunk 5]
```diff
@@ -1,14 +1,95 @@
...
+    }
+  }
+  checkNameExitsOnBlur() {
+    let $this = this;
+    let skillName = $this.skillsModel.name;
+    this.skills.forEach(function (skill) {
+      if (skill['name'] === skillName) {
+        $this.isSkillExists = true;
+        return;
+      }
```
Comment (salilsaini868) [inline web/src/app/admin/skills/skills.component.ts]: need to check duplicate using api methods

File: web/src/app/admin/skills/skills.component.ts [hunk 6]
```diff
@@ -1,14 +1,95 @@
...
+        }
+        this.msg = "deleted successfully";
+      })
+  }
+  editSkills(skills) {
+    this.skillsModel.skillId = skills.skillId;
+    this.skillsModel.name = skills.name;
+    this.skillsModel.description = skills.description;
+    this.isUpdate = true;
+    this.isCreate = false;
```
Comment (salilsaini868) [inline web/src/app/admin/skills/skills.component.ts]: why these conditions are here again?

**Review by salilsaini868 — CHANGES_REQUESTED** [review]

File: web/src/app/admin/qualifications/qualifications.component.ts (hunk 1 above)
Comment (bharathkeerthi) [reply web/src/app/admin/qualifications/qualifications.component.ts]: done

File: web/src/app/admin/skills/skills.component.html (hunk 2 above)
Comment (bharathkeerthi) [reply web/src/app/admin/skills/skills.component.html]: done

File: web/src/app/admin/skills/skills.component.ts (hunk 3 above)
Comment (bharathkeerthi) [reply web/src/app/admin/skills/skills.component.ts]: done

File: web/src/app/admin/skills/skills.component.ts (hunk 4 above)
Comment (bharathkeerthi) [reply web/src/app/admin/skills/skills.component.ts]: done

File: web/src/app/admin/skills/skills.component.ts (hunk 6 above)
Comment (bharathkeerthi) [reply web/src/app/admin/skills/skills.component.ts]: done

File: web/src/app/admin/skills/skills.component.ts (hunk 5 above)
Comment (bharathkeerthi) [reply web/src/app/admin/skills/skills.component.ts]: done

**Review by salilsaini868 — APPROVED** [review]

**Pull request merged**